server/app/views/admin/settings/AdminSettingsIndexView.java

```diff
@@ -297,13 +297,13 @@ private static DivTag renderEnumInput(
 
   private static DivTag renderBoolInput(
       SettingDescription settingDescription, Optional<String> value) {
-    boolean isEnabled = value.map("TRUE"::equals).orElse(false);
+    boolean boolValue = value.map("TRUE"::equals).orElse(false);
```

Comment:
nit:  For the variable name, I actually prefer `isEnabled` or maybe `isOn` because otherwise I find it unintuitive to have `!boolValue`.  Just my preference, though.

Reply:
Yeah I didn't like any of the names I could think of. How about isTrue... it's a little redundant but logically impeccable. 😂